docs/releasing.md

```diff
@@ -24,6 +24,13 @@ Creating a federation v2 release involves the following steps:
    3. `kubefed2 version` (check that the output is as expected)
    4. `tar cvzf kubefed2.tar.gz kubefed2`
    5. `sha256sum kubefed2.tar.gz > kubefed2.tar.gz.sha`
+4. Package the helm chart for release
+   1.  Adjust the default image tag in values.yaml
+   2.  Update the chart version in Chart.yaml
+   3.  Package the chart using `helm package federation-v2`
```

Comment:
As per the kubefed2 instructions, please specify the command to execute and then explain it in brackets, e.g

`helm package federation-v2` (create a tar archive of the chart)

Reply:
Adjusted all instructions